charm/src/iframe/static.ts

```diff
@@ -638,14 +773,14 @@ function onReady(mount, sourceData, libs) {
     return (
       <div className="p-4">
         <animated.div style={props}>
-          <h1 className="text-2xl font-bold">Hello ESM World!</h1>
           <button
             className="mt-2 px-4 py-2 bg-blue-500 text-white rounded"
             onClick={() => setCount(count + 1)}
           >
             Clicks: {count}
           </button>
         </animated.div>
+        <TodoList todos={todos} setTodos={setTodos} />
```

Comment:
Demonstrate multi-component example explicitly and passing `useDoc` functions down the tree.